pom.xml

```diff
@@ -67,14 +67,9 @@
 
 	<dependencies>
 		<dependency>
-			<groupId>org.eclipse.jetty</groupId>
-			<artifactId>jetty-server</artifactId>
-			<version>8.1.8.v20121106</version>
-		</dependency>
```

Comment:
Notice that the dependency to the websocket is a new one ( `org.eclipse.jetty.websocket:websocket-server` instead of `org.eclipse.jetty:jetty-websocket`).
So I am not sure that jetty is needed anymore as the websocket _looks_ like including some server features.

I'll investigate this later on